### PR TITLE
Turning in assignment

### PR DESCRIPTION
**Pull request by karaluton**
- [x] Follow the homework checklist to create and start project.
- [x] **GOAL**: Site Header/Hero
  - [x] Checkpoint: Create markup for Header/Hero
    - [x] Markup: Navbar
      - [x] Markup: Navbar Logo
      - [x] Markup: About Link
      - [x] Markup: Search Link
    - [x] Markup: Hero Title ("Surf & Paddle Co")
    - [x] Markup: Hero Sub Title ("An Ironyard Academy Project Est Fall 2013")
  - [x] Checkpoint: Style for Header/Hero
    - [x] Style: Banner Background
    - [x] Style: Navbar Logo
    - [x] Style: Navbar About
    - [x] Style: Navbar Search
    - [x] Style: Hero Text
    - [x] Style: Hero Subtitle
- [x] **GOAL**: Site Main
  - [x] Checkpoint: Create markup for Site Main
    - [x] Markup: Left Article
      - [x] Markup: Article Heading
      - [x] Markup: Article Author Avatar
      - [x] Markup: Article Publish Date
      - [x] Markup: Article text
      - [x] Markup: Article quote
    - [x] Markup: Right Article Details
      - [x] Markup: Author Image
      - [x] Markup: Author Name
      - [x] Markup: Author Social Links
      - [x] Markup: Ad Image
      - [x] Markup: Ad Link
      - [x] Markup: Sidebar 2nd Article Link
      - [x] Markup: Sidebar 2nd Article Short
  - [x] Checkpoint: Style for Site Main
    - [x] Style: Left Article
      - [x] Style: Article Heading
      - [x] Style: Article Author Avatar
      - [x] Style: Article Publish Date
      - [x] Style: Article text
      - [x] Style: Article quote
    - [x] Style: Right Article Details
      - [x] Style: Author Image
      - [x] Style: Author Name
      - [x] Style: Author Social Links
      - [x] Style: Ad Image
      - [x] Style: Ad Link
      - [x] Style: Sidebar 2nd Article Link
      - [x] Style: Sidebar 2nd Article Short
- [x] **GOAL**: Site Footer
  - [x] Checkpoint: Create markup for Site Footer
    - [x] Markup: Footer heading
    - [x] Markup: Footer Other Posts (Images)
      - [x] Markup: Footer Other Post Item Picture
      - [x] Markup: Footer Other Post Item Title
    - [x] Markup: Footer Navbar
      - [x] Markup: Footer Navbar Logo
      - [x] Markup: Footer Navbar Legal
      - [x] Markup: Footer Navbar Nav Items
  - [x] Checkpoint: Style Site Footer
    - [x] Style: Footer heading
    - [x] Style: Footer Other Posts (Images)
      - [x] Style: Footer Other Post Item Picture
      - [x] Style: Footer Other Post Item Title
    - [x] Style: Footer Navbar
      - [x] Style: Footer Navbar Logo
      - [x] Style: Footer Navbar Legal
      - [x] Style: Footer Navbar Nav Items

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+
+.other-posts__images {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+ .footer-img {
+  width: 100%;
```
Comment (rtablada) [inline app.css:354]: Try setting `height` to `20rem` and then `object-fit: cover`.

**Review by rtablada — CHANGES_REQUESTED** [review]
The background for the footer should still be full width, just limit the text to the same width as your navbar and main content stuff.

screen shot 2016-10-06 at 6 16 56 pm

File: app.css (hunk 1 above)
Comment (rtablada) [reply app.css:354]: This should fix the white area on the 3rd puppy.

Comment (karaluton) [reply app.css:354]: That worked. Thank you!

**Pull request merged**